cores/esp8266/WString.h

```diff
@@ -70,6 +72,9 @@ class String {
         explicit String(double, unsigned char decimalPlaces = 2);
         ~String(void);
 
+        // empty string singleton
+        static constexpr const String& empty = ::__emptyString;
```

Comment:
Please don't call the static member "empty". It should be ok to also call it "emptyString", because it is scoped, or call it something else (emptyStr, nullString, defString, etc)

Reply:
Given the deprecation, can this be implemented the other way around? make this the member the "real" object, and make the global a reference to this one.